src/Illuminate/Container/Container.php

```diff
@@ -785,6 +785,14 @@ public function build($concrete, $parameters = [])
 			throw new BindingResolutionException($message);
 		}
 
+		if (in_array($concrete, $this->buildStack))
+		{
+			$dependency = reset($this->buildStack);
+			$message = "Cyclic reference found while resolving [$concrete] and [$dependency].";
```

Comment:
Check a few lines before this, the same thing is done.

Also if the string is created in the constructor, the line length is 123.